solutions/fully-configurable/main.tf

```diff
@@ -2,19 +2,13 @@
 # Resource Group
 ########################################################################################################################
 locals {
-  # tflint-ignore: terraform_unused_declarations
-  validate_resource_group = (var.existing_secrets_manager_crn == null && var.resource_group_name == null) ? tobool("Resource group name can not be null if existing secrets manager CRN is not set.") : true
-  # tflint-ignore: terraform_unused_declarations
-  validate_event_notifications = (var.existing_event_notifications_instance_crn == null && var.enable_event_notifications) ? tobool("To enable event notifications, an existing event notifications CRN must be set.") : true
-  prefix                       = var.prefix != null ? (var.prefix != "" ? var.prefix : null) : null
+  prefix = var.prefix != null ? (var.prefix != "" ? var.prefix : null) : null
```

Comment:
Can we go with this approach so we don't have to use `try` everywhere
`prefix = var.prefix != null ? trimspace(var.prefix) != "" ? "${var.prefix}-" : "" : ""`

See the way it was done in https://github.com/terraform-ibm-modules/terraform-ibm-scc/blob/main/solutions/fully-configurable/main.tf for reference